index.html

```diff
@@ -211,6 +211,83 @@ <h2 class="class-about-div-title">Skills</h2>
           </li>
         </ul>
       </section>
+
+      
+
+      <footer id="id-footer">
+        <section id="id-contact-form">
+          
+          <div class="class-contact-div">
+            <h1 class="class-sections-titles">Get started</h1>
+            <p class="class-about-description">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+              sed do eiusmod tempor incididunt ut labore et dolore magna.
+            </p>
+          </div>
+
+          <div class="class-contact-div" id="contact-form">
+            <form action="https://formspree.io/f/form" method="post" id="form">
+
+              <div class="class-form-div">
+                <label for="name" class="class-form-label">
+                  <input name="name" type="text" id="name" class="name-email" maxlength="30" required placeholder="your name">
+                </label>
+                
+              </div>
+
+              <div class="class-form-div">
+                <label for="email">
+                  <input type="email" name="email" id="email" class="name-email" required placeholder="your email">
+                </label>
+              </div>
```

Comment:
The name and email fields should be aligned see below 
# original design
![right](https://user-images.githubusercontent.com/58376020/121238313-413c7f00-c898-11eb-9111-1339ca9e86ad.png)

# Your project 
![wrongg](https://user-images.githubusercontent.com/58376020/121238444-67fab580-c898-11eb-8d1f-7d6ab39ec6bd.png)

Reply:
Excuse me but I'm confused about your review. it's a good one but I'm supposed to do the mobile version, not the desktop version. I'm waiting for your answer

Reply:
Hi @Marcraphael12 ,
That's how the original design look on the mobile as the screenshot is taken from the original design

Reply:
> The name and email fields should be aligned see below
> 
> # original design
> ![right](https://user-images.githubusercontent.com/58376020/121238313-413c7f00-c898-11eb-9111-1339ca9e86ad.png)
> 
> # Your project
> ![wrongg](https://user-images.githubusercontent.com/58376020/121238444-67fab580-c898-11eb-8d1f-7d6ab39ec6bd.png)

Good guy! Sorry, we are supposed to do the mobile version of this template, but the pic you show is the desktop version of the template. So I'm very confused about it! Please can you check the template again and let me know your answer? you are the best!